pavement.py

```diff
@@ -10,9 +10,9 @@
 
 
 RUN_PACKAGES = [
-    'sphinx',
-    'sphinx-autobuild',
-    'sphinx_rtd_theme',
+    'sphinx==1.8.5',
```

Comment:
Maybe update to latest Sphinx ?

Reply:
Sphinx dropped Python 2 support in 2018; should I also update to Python 3?

https://github.com/sphinx-doc/sphinx/commit/9412bd76b7f5fdf0adc231ef8130e45bda130164

Reply:
Yes please :)

For Python3 we should use https://github.com/chevah/pythia/

If need help on using pythia let me know or @dumol :)

 pythia is our future py3 own distribution and Mișu is maintaining it